t/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "3.5"
```

Comment:
I think this yaml needs to be removed, we should have a single point of reference for such things (and it should live in dgraph repo)

Reply:
In principle I definitely agree - but the problem is that to run dgo tests, we need a cluster to be up and running.  Ideally someone should be able to checkout the dgo repo and run the tests without also checking out the dgraph repo.  You have any thoughts on how to do that?